src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Person } from '../../types/Person';
+import { PersonRow } from '../PersonRow';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = React.memo(({ people }) => {
+  return (
+    <table className="table">
```

Comment:
Better to check if people.length > 0 (When toggled people link, th of table appearing earlier)